Slim/Menu/BrowseLibrary.pm

```diff
@@ -1754,6 +1754,7 @@ sub _tracks {
 				$_->{'name'}          = $_->{'title'};
 
 				$_->{'type'}          = 'audio';
+				$_->{'db_track'}      = 1;
```

Comment:
Instead of adding another attribute we could simply check for the URL's protocol (`... && item.url.match(/^file:|tmp:/)` or similar.

Reply:
I thought that too, but it doesn't work, as we also need to include remote tracks (Qobuz for example) which have been imported.